src/components/Explore/ObservationsView.js

```diff
@@ -135,6 +135,7 @@ const ObservationsView = ( {
         obsListKey="ExploreObservations"
         onEndReached={fetchNextPage}
         showNoResults={!canFetch || totalResults === 0}
+        hideObsUploadStatus
```

Comment:
This prop is also consumed by the `ObsListItem` [here](https://github.com/inaturalist/iNaturalistReactNative/blob/8ffe02cc98613868141613eb5a4eff3d770a3651/src/components/ObservationsFlashList/ObsPressable.tsx#L77-L78). So, changing like so also removes ObsUploadStatus from the list layout which is not specified in the issue ticket.